File: beginner/pagina02/1038/1038.lua
local prices = {4.00, 4.50, 5.00, 2.00, 1.50}

local code, qnt = io.read():match("(%d+) (%d+)")
code = tonumber(code)
qnt = tonumber(qnt)

local total = prices[code] * qnt

print(string.format("Total: R$ %.2f", total))

File: beginner/__1000__/1038/1038.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);
        double[] prices = {4.00, 4.50, 5.00, 2.00, 1.50};

        int code = in.nextInt();
        int qnt = in.nextInt();

        double valor = prices[code - 1] * qnt;

        System.out.println("Total: R$ " + String.format("%.2f", valor));

        in.close();
    }
}